frontend/src/abs/frontend/typechecker/Typing.jrag

```diff
@@ -93,6 +93,10 @@ aspect Typing {
     eq TypeSynDecl.getType() { 
         return getValue().getType(); 
     }
+
```

Comment:
PartialFunctionDecl instances actually don't vanish anymore, neither do ParFnApp instances which are children of a PartialFunctionDecl.
I guess UnknownType would be the more sensible choice here, although type checking is effectively disabled for instances of both PartialFunctionDecl and ParFnApp.

I'll just try and see that the type-checker throws at me if I change it.

Reply:
Not having PartialFunctionDecl vanish would be good news!  Then you can round-trip pretty-print them (print the PartialFunctionDecl and don't pretty-print the generated functions; check what Lars has been doing with the generated datatype accessor functions).

Reply:
PartialFunctionDecls are printed already. Not printing generated expansions would be trivial, but I don't think we should do that, because then we should also print the generated FnApp as the original ParFnApp, which would be cumbersome.